index.bs

```diff
@@ -1114,7 +1114,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
         :   <dfn>otherUI</dfn>
         ::  OPTIONAL other information used by the [=authenticator=] to inform its UI. For example, this might include the user's
-            {{displayName}}.
+            {{displayName}}. [=otherUI=] is a <dfn>mutable item</dfn> and SHOULD NOT be bound to the [=public key credential source=]
+            in a way that prevents updating it without destroying the [=public key credential source=].
```

Comment:
how about...
```suggestion
            in a way that prevents its being updated.
```
..?

Reply:
I find it hard to parse "its being updated" for some reason. Maybe
```suggestion
            in a way that prevents it from being updated.
```
?

Reply:
I think it still needs the "without destroying the [=public key credential source=]."

Reply:
I feel like this is now a bit ambiguous: does "it" refer to "otherUI" or to "the public key credential source"? Maybe replace `it` with `[=otherUI=]`?

Reply:
Done!